### DISCUSSION TITLE
Could API add-ons be made to merge other stores like GOG and Itch?

### DISCUSSION THREAD

**Discussion by Waifu4Life** [SteamDeckHomebrew/decky-loader#482] [Ideas]
Just curious, could API add-ons be made to merge other stores like GOG and Itch in order to download games and sync achievements?

**Comment by PartyWumpus** [comment]
It's plausible that plugins like emuachievements but for other storefronts could exist if they have suitable APIs.

Edit: Downloading things from those stores would be significantly more complicated, and probably left best to the existing tools like lutris and heroic.
